backend/handlers/base_handler.py

```diff
@@ -14,7 +14,7 @@ class BaseHandler(webapp2.RequestHandler):
     def decorateHeaders(self):
         """Decorates headers for the current request."""
         self.response.headers['Access-Control-Allow-Origin'] = '*'
-        self.response.headers['Access-Control-Allow-Headers'] = 'X-API-KEY, Origin, X-Requested-With, Content-Type, Accept, Access-Control-Request-Method, Authorization'
+        self.response.headers['Access-Control-Allow-Headers'] = 'X-API-KEY, Origin, X-Requested-With, Content-Type, Accept, Access-Control-Request-Method, Authorization, Institution-Authorization'
```

Comment:
Erro: 
File "/home/mayza/workspace/web/backend/handlers/invite_institution_handler.py", line 22, in post
    current_institution = body['currentInstitution']
KeyError: 'currentInstitution'

Quando tento enviar um convite para criação de uma instituição

Reply:
:+1: